manifests/base/eventbus-controller/controller-config.yaml

```diff
@@ -19,6 +19,13 @@ data:
           max_memory_store: -1
           # e.g. 20G. -1 means no limit, Up to 1TB if available
           max_file_store: 1TB
+        streamConfig: |
+          # The default properties of the streams to be created in this JetStream service
+          maxMsgs: 50000
+          maxAge: 168h
+          maxBytes: -1
+          replicas: 3
+          duplicates: 60s
```

Comment:
So, is "duplicates" how long it waits before resending a message that hasn't been acknowledged? or something else?

Reply:
Duration of no duplicate message.